test/integrations/destinations/bloomreach/router/data.ts

```diff
@@ -0,0 +1,220 @@
+import { generateMetadata } from '../../../testUtils';
+import { defaultMockFns } from '../mocks';
+import {
+  destType,
+  destination,
+  traits,
+  properties,
+  headers,
+  endpoint,
+  RouterInstrumentationErrorStatTags,
+} from '../common';
+
+const routerRequest = {
```

Comment:
Please use the new format of test case writing add description and success criteria

Reply:
Mocking is not supported in new format

Reply:
Add two fields description and success criteria that should be fine

Reply:
this is already added